src/cli/qcmp.rs

```diff
@@ -70,7 +70,7 @@ impl Ping {
                 }
             };
 
-            let recv_time = chrono::Utc::now().timestamp_nanos();
+            let recv_time = chrono::Utc::now().timestamp_nanos_opt().unwrap();
```

Comment:
Is there any risk of `timestamp_nanos_opt()` returning `None` ?

Reply:
Only if people are still running Quilkin in a few hundred years. https://docs.rs/chrono/latest/chrono/struct.DateTime.html#method.timestamp_nanos_opt

> An i64 with nanosecond precision can span a range of ~584 years. This function panics on an out of range DateTime.
> The dates that can be represented as nanoseconds are between 1677-09-21T00:12:44.0 and 2262-04-11T23:47:16.854775804.